### ISSUE TITLE
Register launched vThunders with agalaxy

### ISSUE THREAD

**Issue by Cedev** [a10networks/a10-neutron-lbaas#351]
As an orchestration user, I'd like my launched vThunders to be registered with agalaxy.

 - [ ] Add an `'agalaxy': {}` section to vthunder configuration
 - [ ] Register launched vthunders as a `device` with agalaxy. See https://10.48.6.6/agapi/v1/ and https://10.48.6.6/agapi/v1/device/ (or https://10.6.100.34/agapi/v1/device/) for details.

**Comment by mdurrant-b3** [comment]
Does this need to be async?

**Comment by Cedev** [comment]
Yes, it does need to be async. The agalaxy add device call blocks while agalaxy communicates with the device. We don't want to add this long communication time to our device spin-up.

**Comment by Cedev** [comment]
Progress so far: https://gitlab/cshock/agalaxy-client-poc/
